bindings/go/src/fdb/generated.go

```diff
@@ -39,6 +39,11 @@ func int64ToBytes(i int64) []byte {
 	return buf
 }
 
+// Retain temporary external client library copies that are created for enabling multi-threading.
```

Comment:
It doesn't look like https://github.com/apple/foundationdb/pull/8761 was cherrypicked.

Reply:
yes, because I was using the old Go version in okteto.